scripts/generator/graphql_generator/csharp/SDK/IWebCheckout.cs.erb

```diff
@@ -0,0 +1,7 @@
+namespace <%= namespace %>.SDK {
+    using System;
+    
+    interface IWebCheckout {
+        void Start(string checkoutURL, CheckoutSuccessCallback success, CheckoutFailureCallback failure, CheckoutCancelCallback cancelled);
```

Comment:
Again when I see a method named `Start` I expect there to be a matching `Finish` or `Finalize` method.